supplementary_style_guide/style_guidelines/lead-in-sentences.adoc

```diff
@@ -0,0 +1,11 @@
+[[lead-in-sentences]]
+= Lead-in sentences
+
+A lead-in sentenece is the text that directly follows a procedure or prerequiste heading in a task-based module. It is not the module abstract, which describes the goals of the user for the module.
```

Comment:
* sentenece -> sentence
* I'd say "A lead-in sentence in this context is..." or something similar. Because I do believe that the mod docs uses the same terminology

Reply:
@joaedwar Please indicate that a lead-in sentence must be a full sentence to support translation. Translation tools have a hard time with incomplete sentences.

Good: "To do xyz, complete the following steps:"
Bad: "To do xyz:"

(Side note: I always went with the bad option to avoid fluff until one of the translators made me aware of the issue.)

Reply:
+ 1 to that, absolutely! Full sentences over fragments, every single time, Wherever possible (no only in lead-in sentences) :-)